fn/fn_defn.h

```diff
@@ -373,7 +373,7 @@ class [[sus_trivial_abi]] Fn<R(CallArgs...)> : public FnMut<R(CallArgs...)> {
   Fn(F ptr) noexcept : FnMut<R(CallArgs...)>(static_cast<F&&>(ptr)) {}
 
   /// Construction from the output of `sus_bind()`.
-  template <::sus::concepts::callable::LambdaReturns<R, CallArgs...> F>
+  template <::sus::concepts::callable::LambdaReturnsConst<R, CallArgs...> F>
```

Comment:
This is an improved compiler error when making a Fn from sus_bind_mut(), which is invalid. It errors at construction instead of at calling operator().